src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolBase.java

```diff
@@ -77,6 +79,17 @@ public void setUp() {
             .build();
         doReturn(caseObj3).when(retrieveDataService).getCaseById("1234567890123459");
 
+        HashMap<String, JsonNode> caseAttributes = new HashMap<>();
+        caseAttributes.put("REGION", convertValueJsonNode("south-east"));
+        caseAttributes.put("BASE_LOCATION", convertValueJsonNode("London"));
```

Comment:
Drool rule only checks for keys with camel case names like region, baseLocation. These will be ignored by drools.